md/minimizer.py

```diff
@@ -15,15 +15,15 @@ def get_romol_conf(mol):
     return guest_conf/10 # from angstroms to nm
 
 
-def minimize_host_4d(romol, host_system, host_coords, ff, box):
+def minimize_host_4d(mols, host_system, host_coords, ff, box):
     """
     Insert romol into a host system via 4D decoupling under a Langevin thermostat.
```

Comment:
Nitpick: using thermostat should keep system at constant temperature, but minimization should effectively bring system to zero temperature

Reply:
Agreed - this isn't accurate if we set ccs=0 and I need to update the docstring